configure.ac

```diff
@@ -1059,6 +1059,38 @@
         AC_MSG_RESULT(no)
     fi
 
+  # libodp
+    AC_ARG_ENABLE(odp,
+            AS_HELP_STRING([--enable-odp], [Enable ODP support]),,[enable_odp=no])
+    AC_ARG_WITH(libodp_includes,
+            [  --with-libodp-includes=DIR  libpcap include directory],
+            [with_libodp_includes="$withval"],[with_libodp_includes=no])
+    AC_ARG_WITH(libodp_libraries,
+            [  --with-libodp-libraries=DIR    libodp library directory],
+            [with_libodp_libraries="$withval"],[with_libodp_libraries="no"])
+
+    if test "$with_libodp_includes" != "no"; then
+        CPPFLAGS="${CPPFLAGS} -I${with_libodp_includes}"
+    fi
+
+    if test "$with_libodp_libraries" != "no"; then
+        LDFLAGS="${LDFLAGS}  -L${with_libodp_libraries}"
+    fi
+    AC_CHECK_HEADERS([odp_api.h])
+
+    LIBPODP=""
+    AC_CHECK_LIB(odp-linux, odp_init_global,, LIBODP="no")
+    if test "$LIBODP" = "no"; then
+        echo
+        echo "   WARNING!  Optional libodp library not found, go get it"
+        echo "   from http://www.opendataplane.org or your distribution:"
+        echo
+        echo "   Ubuntu: apt-get install libodp-dev"
```

Comment:
Which distro/version has this? Tried Ubuntu 16.04 and 16.10 and both didn't have it.